### ISSUE TITLE
Clarify in Reflow that it primarily applies to body text, not text in containers?

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#3322]
Please see the following question asked by a colleague and my response.

My question to the WG. Should this be used as the basis for an addition to the   Understanding document?

>A question about the [1.4.10: Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html) requirement remains: does the secondary viewport, the search results, also need to reflow? Or, is it OK to truncate the results after 50+ characters are displayed..., as shown in the screenshot?
I think it is not a violation because
>1. Text entry of the search term does NOT get truncated but correctly scrolls right when typing a long string of text (I gave up after typing 99+ character search term)
>1. JAWS (and other magnifiers) reads the full string in the search results, even though it is visually truncated after the 50+ characters. (Most magnifier users I now use magnification with speech)
>1. Visually wrapping the search results would confuse the user that it's an additional search result, especially if it took up more screen real estate to add bullets or some indentation
>1. truncating search results are often designed for "all users" because the results are a type of "hint" that the user uses to determine if they want to explore further (click) or not.
![image](https://github.com/w3c/wcag/assets/14298245/06717738-4c0c-4c51-a543-b44fe2466776)

The screenshot shows a list of search results from an auto-type search input on a magnified screen. Most of the results are long and display an ellipsis at the end of a truncated line.

My response was:
>The SC is intended to facilitate the reading of body text when magnified, the primary concern being that when text is not wrapped, it is awkward and error-prone to scroll back and forth, trying to move from the end of one long line to the start of the next without losing one's place.
With items that are in something like a dropdown or menu, the list of items is already constrained horizontally. It would lead to its own problems of readability to wrap lines in such a situation (since every line is a separate response), hence the truncation. I think your use case is closer to the considerations of text spacing than Reflow.

>It may be a short-coming of the SC that it doesn’t discuss truncation at all, but we have clear precedent in a number of SCs that as long as someone can consume the entire inforrmation by a method. See “Use of ellipses” on this page https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html.

**Comment by scottaohara** [comment]
from a usability standpoint i have mixed feelings about this topic - but i absolutely agree that the 'use of ellipses' from the text spacing understanding doc is quite relevant for reflow, if not also text sizing as well.

**Comment by alastc** [comment]
I think it's a good start, but I'd be careful about terming it 'body text'.

That's not a well defined term, e.g. I'm not sure if that would apply to a table within a body area?

We went with 'content' as the primary scope so that you could have it apply to table cells within a table, but allow for the table as a whole to scroll.

**Comment by mbgower** [comment]
>  I'd be careful about the term 'body text'...That's not a well defined term, e.g. I'm not sure if that would apply to a table within a body area?

@alastc, as you know, we have language stating
> Except for parts of the content which require two-dimensional layout for usage or meaning.

and then in the note state:
> Examples of content which requires two-dimensional layout are images required for understanding (such as maps and diagrams), video, games, presentations, data tables (not individual cells), and interfaces where it is necessary to keep toolbars in view while manipulating content. It is acceptable to provide two-dimensional scrolling for such parts of the content.

So we do already have constraints in the SC, which, if we go this route, allow us to take a subset of 'body text'.

We do not expect horizontally constrained information sections to suddenly get wider (menu items, left navigation),  and I'm not sure that I'd regard the use of horizontal scrollbars in a left nav region to be a failure of Reflow.

As with all [the discussion in WCAG2ICT](https://github.com/w3c/wcag2ict/issues/98) around the relative immaturity of this SC, I think there is opportunity to try to do a better job of containing the SC

**Comment by bruce-usab** [comment]
@scottaohara -- I think question is similar to analogous to [your two code pen examples](https://codepen.io/scottohara/pen/MWZKXop).  From the backlog conversation today, I am more strongly of the opinion neither example fails 1.4.10.  The second one, I would really like to fail since it would be so easy for someone using screen magnification to miss the right-hand side widget.  But since I am okay with the left navbar not interfering (much) with reading the main column of text, I think it is technically similar.  I am looking forward to continuing the discussion!
